src/runtime/standalone.ts

```diff
@@ -20,6 +20,7 @@ export interface CssConfig {
   css: string;
   staticCss: string;
   compilationHash: string;
+  defaults?: string;
```

Comment:
Why it's a string? I thought it should be a result of styles params. Am I missing something?